ivy/functional/frontends/torch/Tensor.py

```diff
@@ -1,4 +1,6 @@
 # local
+import torch.onnx.symbolic_opset9
```

Comment:
I think we shouldn't be importing `torch` in the frontend file, but given that you're not using it anywhere, it may just be something that got automatically added via an IDE code suggestion.

Reply:
Not sure how it added. Will remove.